doc/xcodebuild.txt

```diff
@@ -18,6 +18,9 @@ xcodebuild. By default, xcodebuild.vim passes the `--color` flag to xcpretty.
 ==============================================================================
 CONFIGURATION                             *xcodebuild-configuration*
 
+* Set the runner to use for the xcodebuild command (uses ! by default)
+ let g:xcodebuild_runner = 'VtrSendCommandToRunner'
```

Comment:
:+1: The only other option is there is a bang form of the command, `VtrSendCommandToRunner!` that will open a pane if one is not currently attached. That's my preferred usage for these sort of interactions as it gets close to the "it just works" territory.